client/sandbox/react-nextjs/pages/play/patch.tsx

```diff
@@ -65,6 +68,14 @@ export default function Patch() {
     ]);
   }
 
+  function mergeWithDeepUndef() {
+    db.transact([
+      tx.blocks[singletonIdX].merge({
+        nestedData: { new_key: undefined },
```

Comment:
Will this still work if the user goes offline and then back online? I thought `new_key` would disappear, since we save this to IDB

Reply:
I don't follow.  This is the behavior that was broken

Reply:
Oh, I see!  My original hypothesis was wrong, in fact, we were transforming undefined into null before this code is called.

Reply:
Oh awesome!

Reply:
Right when it hits `Reactor`, in `pushTx`